src/algokit_utils/deploy.py

```diff
@@ -657,6 +660,10 @@ def _deploy_breaking_change(self) -> DeployResponse:
                 "If you want to try deleting and recreating the app then "
                 "re-run with on_schema_break=OnSchemaBreak.ReplaceApp"
             )
+        if self.on_schema_break == OnSchemaBreak.AppendApp:
```

Comment:
Will need to update the tests to include these new deploy scenarios for both OnSchemaBreak and OnUpdate